ATTIC/deployment/local/Tiltfile

```diff
@@ -1,9 +1,9 @@
-# -*- mode: python; -*-
 k8s_yaml(kustomize('.'))
-docker_build('raiinbow/webapp', '../../../apps/webapp',
+docker_build('gcr.io/k8s-staging-apisnoop/webapp', '../../../apps/webapp',
   live_update=[
   fall_back_on(['package.json', 'package-lock.json']),
-  sync('apps/webapp','/src')
+  sync('apps/webapp','/src'),
+  run('touch src/client.js')
```

Comment:
Is live update working?

Reply:
unfortunately, I couldn't get it to work with our svelte app.